internal/jet/literal_expression.go

```diff
@@ -326,6 +326,102 @@ func formatNanoseconds(nanoseconds ...time.Duration) string {
 	return ""
 }
 
+// ---------------------------------------------------//
+type rangeLiteral struct {
+	rangeInterfaceImpl
+	literalExpressionImpl
+}
+
+func RawRange(raw string, namedArgs ...map[string]interface{}) RangeExpression {
+	return RangeExp(Raw(raw, namedArgs...))
+}
+
+func NumRange(lowNum, highNum *int, bounds string) RangeExpression {
```

Comment:
- `NumRange` should accept `NumericExpression`, since postgres does allow expression to appear as a parameter. For instance:   
```sql
select numrange(MIN(actor.actor_id), 100)
from dvds.actor;
```
- To avoid dealing with pointers and to represent infinities we need to define a new type `PlusInfinity` and `MinusInfinity`, similary to existing [`NULL` type](https://github.com/go-jet/jet/blob/e03773a79ec5e78fdc2ddd9ceb8057dd0d58c0c6/internal/jet/literal_expression.go#L333). The type would be `Expression`, but we can then convert it to `IntegerExpressoin` or `FloatExpression`, with `IntExp(PlusInfinity)`, `FloatExp(MinusInfinity)`
- according to postgres `bounds` can be omitted, we can simulate that with variadic arguments `,bounds... string)`